demo/index.html

```diff
@@ -189,10 +199,11 @@ <h2>Have Fun</h2>
 			<div id="pixi-container">
 				<script>
 PIXI.settings.RESOLUTION = 2;
-let renderer = PIXI.autoDetectRenderer(600, 3100);
+let renderer = PIXI.autoDetectRenderer(600, 3425);
 renderer.backgroundColor = 0x333333;
 document.getElementById("pixi-container").appendChild(renderer.view);
 let stage = new PIXI.Container();
+const MultiStyleText = this.pixiMultistyleText;
```

Comment:
Why is this needed? This is making the demo to fail on my machine. 

Not sure why the name would become `pixiMultistyleText`. The name used in microbundle is `MultiStyleText`.

Reply:
ah sorry,I had to add it to get it to load on linux. I will remove it now